frontend/src/components/Issue/IssueDetailLayout.vue

```diff
@@ -731,18 +772,36 @@ export default defineComponent({
           );
         case "bb.task.database.restore":
           return "";
+        case "bb.task.database.schema.update.ghost.sync":
+        case "bb.task.database.schema.update.ghost.cutover":
+        case "bb.task.database.schema.update.ghost.drop-original-table":
+          return ""; // should never reach here
       }
     };
 
     const isTenantDeployMode = computed((): boolean => {
+      if (project.value.tenantMode !== "TENANT") return false;
+      return (
+        props.issue.type === "bb.issue.database.schema.update" ||
+        props.issue.type === "bb.issue.database.data.update"
+      );
+    });
+
+    const isGhostMode = computed((): boolean => {
+      if (!isDev()) return false;
+
+      return props.issue.type === "bb.issue.database.schema.update.ghost";
+    });
+
+    const isGhostSyncTaskSelected = computed((): boolean => {
       return (
-        props.issue.type === "bb.issue.database.schema.update" &&
-        project.value.tenantMode === "TENANT"
+        selectedTask.value.type === "bb.task.database.schema.update.ghost.sync"
```

Comment:
nit: remove the extra brackets.

Reply:
it's added by prettier code formatter since it will be too long (> 80 chars) if we combined these two lines.

Reply:
gotit, then let's follow prettier then. Thanks!